zerver/tests/test_subs.py

```diff
@@ -3672,8 +3677,8 @@ def test_users_getting_remove_peer_event(self) -> None:
                         get_client("website"),
                     )
 
-        self.assert_length(query_count, 28)
-        self.assert_length(cache_count, 4)
+        self.assert_length(query_count, 49)
+        self.assert_length(cache_count, 14)
```

Comment:
These increases in tested query counts are almost certainly a new performance problem.

Reply:
These increases in query I think are due to the ```internal_send_private_message``` being called for each user. As told by Steve in https://chat.zulip.org/#narrow/stream/92-learning/topic/Stop.20unsubscribing/near/1135159 and I also find that there is no alternative efficient way to send private message in bulk.

Reply:
I have mentioned in the commit the reason for increased query counts.